examples/python/ghz.py

```diff
@@ -65,7 +64,7 @@
 # Create a GHZ state
 qc.h(q[0])
 for i in range(4):
-    qc.cx(q[i], q[i+1])
+    qc.cx(q[i], q[i + 1])
```

Comment:
Good catch! However, due to the nature of the SDK, would it be possible not to consider [E226](http://pep8.readthedocs.io/en/release-1.7.x/intro.html#error-codes) (`missing whitespace around arithmetic operator`) an error to be fixed? There is indeed a bit of discussion around it, and the style-checkers based on `pycodestyle` usually do not enforce it by default: 

> In the default configuration, the checks E121, E123, E126, **E133**, **E226**, E241, E242 and E704 are ignored because they are not rules unanimously accepted, and PEP 8 does not enforce them. Use switch --hang- closing to report E133 instead of E123.